packages/shopping-cart/src/types.ts

```diff
@@ -564,7 +564,8 @@ export interface ResponseCartProductExtra {
 
 	// Marketplace properties
 	is_marketplace_product?: boolean;
-	plugin_slug?: boolean;
+	product_slug?: boolean;
```

Comment:
```suggestion
	product_slug?: string;
```

This should be string, right? I think the current version is wrong.

Reply:
Ooops, yes it should, I didn't notice that one!